observability/test/manual_deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+cd "$(dirname "$0")"
+
+export PROJECT=`gcloud config get-value project`
+export PROJNUM=`gcloud projects describe $PROJECT --format="value(projectNumber)"`
+export SERVER_IMAGE=gcr.io/microsvcs-testing/grpc-observability/testing/integration-dev-go:1.53.0-dev
+export CLIENT_IMAGE=gcr.io/microsvcs-testing/grpc-observability/testing/integration-dev-go:1.53.0-dev
```

Comment:
Please make sure that multiple other people on the team have access to this project.

Reply:
Yes most of the gcp observability team is on that project.